nearlib/nearbase.js

```diff
@@ -0,0 +1,52 @@
+const ed25519 = require('ed25519');
+const bs58 = require('bs58');
+
+/**
+ * Base class for near libraries. Contains common functionality for communicating with near. 
+ */
+class NearBase {
```

Comment:
I think it's better to just name it `NearClient` and use in other classes instead of extending.

Reply:
done